packages/core/src/browser/progress-status-bar-item.ts

```diff
@@ -73,7 +73,7 @@ export class ProgressStatusBarItem implements ProgressClient {
             this.statusBar.removeElement(this.id);
             return;
         }
-        const text = `$(refresh~spin) ${message}`;
+        const text = `$(codicon-sync~spin) ${message}`;
```

Comment:
This reminds me of the following issue https://github.com/eclipse-theia/theia/issues/9768, perhaps it's something we can easily implement in the future.